lib/src/page_list_viewport_gestures.dart

```diff
@@ -288,17 +303,35 @@ class _PageListViewportGesturesState extends State<PageListViewportGestures> wit
   }
 }
 
-class DeprecatedPanAndScaleVelocityTracker {
-  /// The maximum velocity a gesture can have and still be considered a viewport
-  /// re-adjustment. Used in tandem with the [kViewportReAdjustmentMinTranslationDistance]
-  /// to determine if the user is re-adjusting the viewport. Viewport re-adjustments
-  /// result in the momentum simulation to be aborted.
-  static double kViewportReAdjustmentMaxVelocity = 600;
+// statically defined class with double constant values which parametrize
+// charateristic translation distances and velocities
+// distances are tiny | small | large
+// speeds are slow | normal | fast
+class KViewportScaleThresholds {
```

Comment:
Don't prepend "K" or "k" on the front of classes, functions, or properties. This is a superfluous style from the very early days of Flutter. I see that someone already started this practice in here. I must have missed that in code review.